stl/inc/thread

```diff
@@ -364,6 +364,10 @@ public:
         return thread::hardware_concurrency();
     }
 
+    _NODISCARD native_handle_type native_handle() {

```

Comment:
We should strengthen `noexcept` here and in `thread`, since we're simply returning a `void*` in both cases:
```suggestion
    _NODISCARD native_handle_type native_handle() noexcept /* strengthened */ {
```

Reply:
Makes sense. Added `noexcept` here and in `thread`.